### PR TITLE
[Fadipe Temitope] - Sage - Overview of NFTs on the Celo Platform

### PR DESCRIPTION
**Pull request by Toprule**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [ ] I've thoroughly explained the code, with inline code comments and text explanations.
- [ ] I've run a linter & prettifier to format my code blocks.
- [ ] I've run the tutorial to test it and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

File: src/data-tutorials/users.tsx [hunk 1]
```diff
@@ -1563,6 +1563,13 @@ const Users: User[] = [
     website: 'blog/tutorials/potential-growth-of-blockchain-and-its-use-cases',
     tags: ['celosage', 'beginner'],
   },
+  {
+    title: 'Overview of NFTs on the Celo Platform',
+    description: 'In this tutorial, we explore NFTs and what NFTs platforms are on Celo',
+    preview: require('./showcase/beginner/overview-of-NFTs-on-the-Celo-platform.png'),
```
Comment (developerkunal) [inline src/data-tutorials/users.tsx:1569]: Image path has issues kindly fix it.

Comment (developerkunal) [reply src/data-tutorials/users.tsx:1569]: require('./showcase/beginner/overview-of-NFTs-on-the-Celo-Platform.png')

It should be this.

File: blog/02-15-2023-overview-of-nfts-on-the-celo-platform [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+### Ariswap
+Ariswap is an NFT marketplace built on the Celo network. You can discover, sell and collect unique art pieces on this marketplace. Ariswap allows artists to create collections and upload their works which can be video, image, audio, or 3D art. You can choose to make them static NFTs or dynamic ones with stats, properties, and unlockables.
+
+### Tatum
+Tatum allows users to mint NFTs at express speed on multiple blockchains including Celo. This is made possible using their API key, with Tatum express feature you can mint unlimited NFT at instant speed. Gas fees are deducted from your monthly payment plan on their platform.  
+
+## Conclusion 
+NFTs will continue to see more adoption and even greater use cases. For now, PFP NFTs like Bored Ape and CryptoPunk get a massive shine. Artists like Beeple continue to show other artists how profitable selling their art as NFTs can be. Music, Sport, and more utility NFTs will likely increase in the near future. Celo blockchain network already hosts multiple unique marketplaces where crowdfunding and project support through NFTs are possible.
+
+## About the Author
```
Comment (developerkunal) [inline blog/02-15-2023-overview-of-nfts-on-the-celo-platform:79]: Add spaces after each headlines

**Comment by developerkunal** [comment]
Your file needs to be .md extension

**Comment by Toprule** [comment]
> Your file needs to be .md extension

What particular file?

**Review by viral-sangani — APPROVED** [review]

**Pull request merged**